aldryn_events/cms_wizards.py

```diff
@@ -79,6 +71,11 @@ def __init__(self, **kwargs):
             'Date Acceptable Formats: 2015-11-30, 11/30/2015, 11/30/15'
         )
 
+        self.fields['description'] = forms.CharField(
```

Comment:
ideally we just rename this field to something else to avoid conflict with the description field, that way we don't have to declare it here.